src/__tests__/e2e/translation-e2e-simple.test.ts

```
@@ -0,0 +1,274 @@
+    // Clean output directory
+    await fs.rm(outputDir, { recursive: true, force: true });
+    await fs.mkdir(outputDir, { recursive: true });
+    
+    // Set up FileService mock
+    FileService.setTestInvokeOverride(async (command, args) => {
+      switch (command) {
+        case 'read_text_file':
+          return await fs.readFile(args?.path as string, 'utf-8');
+        
+        case 'write_text_file':
+          const filePath = args?.path as string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)